### PR TITLE
Update contacts page to improve clarity

### PR DESCRIPTION
**Pull request by jrha**
Based on some feedback and experiences here.

More buttons... people like buttons.

### CODE REVIEW

**Comment by jrha** [comment]
I expect tests to fail due to broken links elsewhere.

File: contacts/index.md [hunk 1]
```diff
@@ -22,22 +26,23 @@ If you're not sure which repository to open the issue again (there are a lot) pl
...
-* Computer, smartphones: You can use [vidyoportal]. The room name is `Standup_Meeting`.
-    * You need to login as guest (name is free, use something informational!).
+* **Computers and smartphones:** connect with the Vidyo client, the room name is `Standup_Meeting`.
+    * You need to login as guest (name is free-form, please use something helpful).
     * Note that by default your Vidyo client starts with both audio and video muted
-* Phone bridges: dial phone number of the selected bridge, enter 109296756 as the extension (both numbers are generally followed by #)
+* **Phone bridge:** dial phone number of the selected bridge, enter 109296756 as the extension (both numbers are generally followed by #)
     * See phone bridge [list and instructions]. Please note: if your local number does not work, try using the CERN number.
 
-[vidyoportal]:https://vidyoportal.cern.ch/flex.html?roomdirect.html&key=V8Dpb6amjXWr
```
Comment (stdweird) [inline contacts/index.md]: hey, i use this 😄

File: contacts/index.md [hunk 2]
```diff
@@ -22,22 +26,23 @@ If you're not sure which repository to open the issue again (there are a lot) pl
...
-Development progress is reviewed at a weekly standup meeting held by Vidyo every Thursday at 1:15 pm CET. Connection information is:
+Development progress is reviewed at a weekly Standup meeting held on Vidyo every Thursday at 13:15 CET.
+
+#### Connection information
+
+<a class='btn btn-primary' href='https://vidyoportal.cern.ch/flex.html?roomdirect.html&key=V8Dpb6amjXWr'>Connect with Vidyo</a>
 
-* Computer, smartphones: You can use [vidyoportal]. The room name is `Standup_Meeting`.
-    * You need to login as guest (name is free, use something informational!).
+* **Computers and smartphones:** connect with the Vidyo client, the room name is `Standup_Meeting`.
```
Comment (stdweird) [inline contacts/index.md]: an url to said client is nice

Comment (jrha) [inline contacts/index.md]: The portal will redirect you to the client download if it's not running.

File: contacts/index.md (hunk 1 above)
Comment (jrha) [inline contacts/index.md]: It's the button now!

Comment (stdweird) [inline contacts/index.md]: ahhh, i see. butttonnnnnsssss!!!!!!!!

**Comment by stdweird** [comment]
test fail with broken external links on other pages. merging

**Pull request merged**